pkg/cmd/schedule.go

```diff
@@ -61,6 +64,10 @@ var scdCmd = &cobra.Command{
 		pw_lwt = viper.GetFloat64("pw_lwt")
 		pw_upt = viper.GetFloat64("pw_upt")
 		pw_batt_reserve = viper.GetFloat64("pw_batt_reserve")
+		s_cache_forecast = viper.GetBool("cache_forecast")
+		s_cache_file_prefix = viper.GetString("cache_file_prefix")
+		s_cache_time = viper.GetInt32("cache_time")
```

Comment:
in this file modifing the `checkScheduleschedule` function please add a check to verify `s_cache_time` is >=0